doc/client_lua_api.md

```diff
@@ -754,6 +759,22 @@ Call these functions only at load time!
     * returns reference to mod private `StorageRef`
     * must be called during mod load time
 
+### Mod channels
+You can find mod channels communication scheme in `docs/mod channels.png`.
+
+* `minetest.mod_channel_join(channel_name)`
+    * Client joins channel `channel_name`. If channel has no member it's created. You
+      should listen from incoming messages with `minetest.register_on_modchannel_message`
+      call to receive incoming messages. Warning, this function is asynchronous,
+    * You should use a minetest.after(2, function() ... end) at least, to permit server
```

Comment:
this sentence is confusing
should `minetest.mod_channel_join()` only be called with delay? why?

Reply:
no, you can use channel only when join is okay, but using a ModChannelRef object can permit to handle this use case, i will work on it